File: Ruby/Stakeholder.rb
require('./Proyecto.rb');

public
class Stakeholder

	public
	def getId()
		# Not yet implemented
	end

	# @ParamType aId int
	# @ReturnType void
	public
	def setId(aId)
		@_id = aId
	end

	public
	def stakeholder()
		# Not yet implemented
	end

	# @ReturnType String
	public
	def getNombre()
		return @_nombre
	end

	# @ParamType aNombre String
	# @ReturnType void
	public
	def setNombre(aNombre)
		@_nombre = aNombre
	end

	# @ReturnType float
	public
	def getPresupuesto()
		return @_presupuesto
	end

	# @ParamType aPresupuesto float
	# @ReturnType void
	public
	def setPresupuesto(aPresupuesto)
		@_presupuesto = aPresupuesto
	end

	public
	def initialize()
		# @AttributeType int
		@_id
		# @AttributeType String
		@_nombre
		# @AttributeType float
		@_presupuesto
		# @AttributeType Proyecto
		# # @AssociationType Proyecto
		# # @AssociationMultiplicity 1
		@_tiene

		# Not yet implemented
	end
end

File: C#/Stakeholder.cs
using System;

public class Stakeholder {
	private int id;
	private String nombre;
	private float presupuesto;

	public GetId() {
		throw new System.NotImplementedException("Not implemented");
	}
	public void SetId(ref int id) {
		this.id = id;
	}
	public Stakeholder() {
		throw new System.NotImplementedException("Not implemented");
	}
	public String GetNombre() {
		return this.nombre;
	}
	public void SetNombre(ref String nombre) {
		this.nombre = nombre;
	}
	public float GetPresupuesto() {
		return this.presupuesto;
	}
	public void SetPresupuesto(ref float presupuesto) {
		this.presupuesto = presupuesto;
	}

	private Proyecto tiene;

}
